### DISCUSSION TITLE
Allow backend pixel coordinates to be float?

### DISCUSSION THREAD

**Discussion by DaMilyutin** [plotters-rs/plotters#326] [Ideas]
I've seen some papers proposing float coordinates for backend pixels. Ex. For antialiased lines.
If we allow backend coordinates to be float (ex. f64) we could have more control on plotting.
Maybe, even more nice looking plots?

This will require some updates of algorithms for primitives, however.

I'm not sure if it is best practice.
What is best option?

**Comment by 38** [comment]
Hi this is on our to do list already. We are going to clean up those legacy assumption of pixel based devices in the next major release. See related issue at #128
